sagemaker_neo_compilation_jobs/deploy_huggingface_model_on_Inf1_instance/inf1_bert_compile_and_deploy.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
```

Comment:
<p>Please clear all notebook outputs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Hello, and thank you for the quick review. </p><p>Will do that across the board, and also remove this cell since requirements and inference code are mentioned later.</p>